### ISSUE TITLE
fftw crash

### ISSUE THREAD

**Issue by AbsurdlySuspicious** [wwmm/easyeffects#1189]
### EasyEffects Version

249311c4

### What package are you using?

AUR (easyeffects-git)

### Distribution

Arch linux

### Describe the bug

Initially reported in #1172 ([comment](https://github.com/wwmm/easyeffects/issues/1172#issuecomment-934396112))

Since this crash is unrelated here's separate issue.
Only modules enabled are equalizer and crystalizer, so it shouldn't be related to multiple plugins using fftw. This can be a regression since I haven't encountered this crash on 6.1.0 (I'm not sure about later versions before 6.1.3) 

Trace with !strip: https://gist.github.com/AbsurdlySuspicious/d460f394665cd8c92ef761641cba9a2f
I can send a dump too if this helps

**Comment by wwmm** [comment]
>This seems an issue with vectors in Crystalizer. 

Unfortunately it is worse than that. I tried to debug this problem in the past and it can happen on the convolver or any other code that directly or indirectly calls fftw. Part of the problem I solved by ensuring that the thread initializing fftw was the same destroying it. But there is something more  subtle in the fftw usage that I could not figure out yet.
